src/test/java/com/bookJourney/springboot/mocks/ProfileDTOMock.java

```diff
@@ -0,0 +1,17 @@
+package com.bookJourney.springboot.mocks;
+
+import com.bookJourney.springboot.dto.ProfileDTO;
+
+import java.time.LocalDate;
+
+import static com.bookJourney.springboot.mocks.MockedValues.*;
+
+public class ProfileDTOMock {
+
+    private ProfileDTOMock() {
+    }
+    
+    public static ProfileDTO getProfileDTO() {
+        return new ProfileDTO(USERNAME, FIRST_NAME, LAST_NAME, LocalDate.now());
```

Comment:
datę proponuję zastąpić też mockedValue statycznie ustawioną (LocalDate.of(...))